src/Servers/Kestrel/Core/src/Internal/Http/HttpRequestHeaders.cs

```diff
@@ -243,11 +243,11 @@ internal Enumerator(HttpRequestHeaders collection)
                 : default;
         }
 
-        public KeyValuePair<string, StringValues> Current => _current;
+        public readonly KeyValuePair<string, StringValues> Current => _current;
```

Comment:
Undo?

Reply:
Reason?

Reply:
https://github.com/dotnet/aspnetcore/pull/39637#issuecomment-1017114378

Reply:
The comment is about backwards compat which doesn't apply here because this type is internal. And about auto-properties which doesn't apply because these aren't auto-properties.

You're going to need to provide more detail.

Reply:
It was about this https://github.com/dotnet/aspnetcore/pull/39637#issuecomment-1017114378